templates/email/messages/order_cancelled_en.html

```diff
@@ -0,0 +1,22 @@
+<p>Dear customer, we have received your cancellation. The invoice, application and agreement related to the berth are cancelled as of {{ order.rejectedAt }}.</p>
+
+<h3>Object</h3>
```

Comment:
`Object` sounds a bit odd in English. Not sure what would  be the best but it sounds a bit strange.

Reply:
Should this be Paikka / Plats / Berth?

Reply:
It's `Kohde` in Finnish, there's `Paikka` a couple of lines below. What do you suggest @TatuArvela would be the right translation for `Kohde` in English? I also got translated `Område` in Swedish, do you think that's right?

Reply:
I think we're using "Paikka / Plats / Place" as the generic term when talking about both berths and winter storage places, so we could consider using that.

If using "Kohde", I'd say "Location" would be a good English translation. "Område" means "area", but can make sense in this context I think.

Reply:
Fixed